### DISCUSSION TITLE
Repository Licensing

### DISCUSSION THREAD

**Discussion by xlxs4** [julia-users-berlin/julia-users-berlin.github.io#46] [Q&A] [answered]
Hi, I was wondering whether we can use part of your website as inspiration for the Greek local community organization (https://github.com/julia-lang-athens)

**Comment by theogf** [comment, marked as answer]
Absotulely go for it! There is still some improvement needed on our side though 😅

**Comment by pfitzseb** [comment]
Probably worth adding a LICENSE file (no idea how to do that correctly -- in theory we'd need the approval of all contributors or something?).

**Reply by daveh19** [reply]
This is true. There are only 16 contributors in the entire history. You have my permission. Shall we just agree on a specific licence and ping everybody to do a one time agreement?

**Reply by daveh19** [reply]
Wait, I just checked - we already have one it's just on the page rather than in the repo....
http://creativecommons.org/licenses/by-sa/4.0/ linked from the bottom of the landing page.

I had a feeling we tackled this early.

**Reply by amirallami-code** [reply]
> Probably worth adding a LICENSE file (no idea how to do that correctly -- in theory we'd need the approval of all contributors or something?).

Hi @pfitzseb , if you want to know more about License and licensing you can read this guide, I cover important information about licensing [Link](https://github.com/amirallami-code/license-guide)
